zarrdataset/_zarrdataset.py

```diff
@@ -49,6 +50,13 @@ def close(self, *args, **kwargs):
     PYTORCH_SUPPORT = False
 
 
+def get_ddp_info():
+    if dist.is_available() and dist.is_initialized():
```

Comment:
This function should also consider the case when pytorch is not installed in the environment.
I recommend using `PYTORCH_SUPPORT` to determine the output of this function, like:
```
if PYTORCH_SUPPORT:
    if dist.is_available() and dist.is_initialized():
        return dist.get_rank(), dist.get_world_size()
return 0, 1